src/main/java/com/minecolonies/coremod/client/gui/WindowHutDeliveryman.java

```diff
@@ -0,0 +1,72 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.views.SwitchView;
+import com.minecolonies.coremod.colony.buildings.BuildingDeliveryman;
+import com.minecolonies.coremod.lib.Constants;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Window for the farmer hut.
+ */
+public class WindowHutDeliveryman extends AbstractWindowWorkerBuilding<BuildingDeliveryman.View> implements Button.Handler
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "Handler" is implemented by a super class; there is no need to implement it here. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1939)